init.ps1

```diff
@@ -81,6 +95,78 @@ function CreateDirectory($message, $path){
     New-Item -Force -ItemType Directory -Path $path | Out-Null
 }
 
+function Copy-GithubRepository

```

Comment:
I wonder if we went with a nuget approach here? This way we could have the script search for packages by a predetermined tag and list them out as part of the wizard.